app/assess/views/filters.py

```diff
@@ -8,6 +8,7 @@ def status_to_human(status: str):
         "SUBMITTED": "Submitted",
         "COMPLETED": "Completed",
         "FLAGGED": "Flagged",
+        "ASSESSMENT_COMPLETE": "Assessment complete"
```

Comment:
Can't we just use COMPLETED? Is there a need for this logic?

Reply:
Bug was raised to change it to 'Assessment complete'. Although I had it as 'Completed' originally

Reply:
But can't we keep the key the same and just change the value? As in change L9 to "Assessment complete"

Reply:
Wasn't sure if the status "Completed was being used somewhere else so added new line ?

Reply:
Yeah good point I think its being used elsewhere on criterias like you said:
https://github.com/communitiesuk/funding-service-design-assessment/blob/c783013bd61b086b39d721e1e800985d00b09380/app/assess/templates/macros/criteria_element.html#L18

Should we move the logic inside the banner_summary macro so it applies to all pages instead of the task list?  
Can we also check that if the status is completed the banner state is shared across subpages i.e sub criteria?

Reply:
Yes, we could move the logic to banner_summary. 
Here is the prototype: https://fsd-pre-award.herokuapp.com/s26-assessment/assessment-overview. I think the value should be "Complete" for "COMPLETED" instead of "Completed"